### PR TITLE
Adding a second contig to Dataflow tests.

### PR DESCRIPTION
**Pull request by davidadamsphd**
Now, we many Dataflow tests have reads, variants, and reference bases on contig "2" as well as "1". I also added a useful utility DoFn: PrintCollection.

### CODE REVIEW

File: src/test/java/org/broadinstitute/hellbender/engine/dataflow/ReadsPreprocessingPipelineTestData.java [hunk 1]
```diff
@@ -81,59 +87,70 @@ public ReadsPreprocessingPipelineTestData(Class<?> clazz) {
...
-                KV.of(reads.get(3), getBases(reads.get(3).getStart(), reads.get(3).getEnd()))
+                KV.of(reads.get(0), getBases("1", reads.get(0).getStart(), reads.get(0).getEnd())),
+                KV.of(reads.get(1), getBases("1", reads.get(1).getStart(), reads.get(1).getEnd())),
+                KV.of(reads.get(2), getBases("1", reads.get(2).getStart(), reads.get(2).getEnd())),
+                KV.of(reads.get(3), getBases("1", reads.get(3).getStart(), reads.get(3).getEnd())),
+                KV.of(reads.get(4), getBases("2", reads.get(4).getStart(), reads.get(4).getEnd()))
         );
 
         variants = Lists.newArrayList(
                 new SkeletonVariant(new SimpleInterval("1", 170, 180), true, false, new UUID(1001, 1001)),
```
Comment (lbergelson) [inline src/test/java/org/broadinstitute/hellbender/engine/dataflow/ReadsPreprocessingPipelineTestData.java]: These reads are still brittle to shard size issues aren't they?

Comment (lbergelson) [inline src/test/java/org/broadinstitute/hellbender/engine/dataflow/ReadsPreprocessingPipelineTestData.java]: Verified, they are very brittle.  Changing `REFERENCE_SHARD_SIZE` to 99999 starts causes test failures.

**Comment by lbergelson** [comment]
The class comment on `ReadsPreprocessingPipelineTestData` needs updating.

**Comment by lbergelson** [comment]
Other than the class comments this looks good.  (Feel free to merge if those are the only changes you decide to make.) 

I think we do need to fix the brittleness in the tests with respect to the reference shard and variant shard sizes.  (I saw this was done in some classes, but not all of them yet.)   I think it's fine not to deal with really impractical cases like like shard size -10 or 1, but I think it should be robust to plausible changes.   Could you either do that in this  pull request or create a new ticket.

File: src/test/java/org/broadinstitute/hellbender/engine/dataflow/ReadsPreprocessingPipelineTestData.java (hunk 1 above)
Comment (davidadamsphd) [inline src/test/java/org/broadinstitute/hellbender/engine/dataflow/ReadsPreprocessingPipelineTestData.java]: *that* fix will be a different PR :)

**Comment by lbergelson** [comment]
@davidadamsphd back to you
